src/components/ListOrgs.vue

```diff
@@ -50,6 +50,14 @@
               </span>
             </div>
           </div>
+          <div v-if="activeOrgType === 'groups'" class="mx-2">
```

Comment:
Instead of using a string literal, let's use the constant `ORG_TYPES.GROUPS` that we've recently introduced for this purpose

Reply:
ah, great. didn't know we had a const for that. thanks for the suggestion.